tests/test_models.py

```diff
@@ -29,3 +29,12 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+def test_daily_stddev():
+    from inflammation.models import daily_mean
```

Comment:
Wrong function import
```suggestion
    from inflammation.models import daily_stddev
```